.github/workflows/osx-installer.yml

```diff
@@ -116,6 +119,26 @@ jobs:
           name: osx-installer
           path: osx-installer
 
+  test-install:
+    needs: build
+    runs-on: macos-latest
+    steps:
+      - name: Download test installer
+        uses: actions/download-artifact@v2
+        with:
+          name: osx-installer
+          path: osx-installer
+      - name: Run test
+        run: |
+          set -x
+
+          # Install the git package
+          pkgpath=$(find ./osx-installer/*.pkg)
+          sudo installer -verbose -pkg $pkgpath -target /
+
+          # Show build options
+          git version --build-options
```

Comment:
We will need to manually verify this output, but that's fine.

Reply:
couldn't we automate this?  Have the build step write an additional artifact with the OID and then test it here.
Might not be worth the trouble (you know my love of all things yml...)

Reply:
Yes! Should be pretty straightforward to implement, and it makes it clearer when something goes wrong.

Reply:
Indeed, you can make it an output variable. It is slightly convoluted because you will have to have _two_ output variables (the `test-install` job can only access the outputs of another _job_, not of another job's _step_), see e.g. https://github.com/git/git/blob/v2.34.1/.github/workflows/main.yml#L36, https://github.com/git/git/blob/v2.34.1/.github/workflows/main.yml#L11-L12, and https://github.com/git/git/blob/v2.34.1/.github/workflows/main.yml#L81.

Reply:
It took a bit of experimentation, but this is now working!